js/src/wp-seo-term-scraper.js

```diff
@@ -299,5 +366,26 @@ window.yoastHideMarkers = true;
 		jQuery( document ).on( "wp-collapse-menu wp-menu-state-set", function() {
 			app.snippetPreview.handleWindowResizing();
 		} );
+
+		const snippetEditorData = {
+			title: termScraper.getSnippetTitle(),
+			slug: termScraper.getSnippetCite(),
+			description: termScraper.getSnippetMeta(),
+		};
+
+		store.dispatch( updateData( snippetEditorData ) );
+
+		store.subscribe( () => {
```

Comment:
Since store.subscribe is called on every store change, we should implement some equality checking `state.snippetEditor.data === prevSnippetEditorData`.

Reply:
I do not agree with this approach. Because of the nature of `store.subscribe` I think it would be wise to always to equality checking in the subscribe function itself, instead of relying on some code down the road somewhere. Should be discussed with @atimmer.